engine/src/main/java/org/terasology/engine/network/internal/ServerImpl.java

```diff
@@ -357,7 +360,8 @@ private void updateEntity(NetData.UpdateEntityMessage updateEntity) {
         if (currentEntity.exists()) {
             NetworkComponent netComp = currentEntity.getComponent(NetworkComponent.class);
             if (netComp == null) {
-                logger.error("Updating entity with no network component: {}, expected netId {}", currentEntity, updateEntity.getNetId());
+                logger.atError().addArgument(() -> currentEntity).addArgument(() -> updateEntity.getNetId()).
+                        log("Updating entity with no network component: {}, expected netId {}");
```

Comment:
`updateEntity.getNetId()` called before and later-on, so create and reference local variable